examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/ui/theme/Type.kt

```diff
@@ -2,19 +2,50 @@ package com.revenuecat.paywallstester.ui.theme
 
 import androidx.compose.material3.Typography
 import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.Font
 import androidx.compose.ui.text.font.FontFamily
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.unit.sp
+import com.revenuecat.paywallstester.R
+
+val superMarioFontFamily =
+    FontFamily(
+        Font(
+            R.font.super_mario,
+        )
+    )
 
 // Set of Material typography styles to start with
 val Typography = Typography(
     bodyLarge = TextStyle(
-        fontFamily = FontFamily.Default,
+        fontFamily = superMarioFontFamily,
         fontWeight = FontWeight.Normal,
         fontSize = 16.sp,
         lineHeight = 24.sp,
         letterSpacing = 0.5.sp,
     ),
+    displaySmall = TextStyle(
+        fontFamily = superMarioFontFamily,
+        fontWeight = FontWeight.Normal,
+        fontSize = 22.sp,
+        lineHeight = 28.sp,
+        letterSpacing = 0.sp
+    ),
+    titleLarge = TextStyle(
+        fontFamily = superMarioFontFamily,
+        fontWeight = FontWeight.Normal,
+        fontSize = 22.sp,
+        lineHeight = 28.sp,
+        letterSpacing = 0.sp
+    ),
+    titleMedium = TextStyle(
```

Comment:
This might be a bit confusing... Basically the developer needs to override the specific styles we use within the paywall in order to use their typography everywhere, so maybe we should keep a list of these somewhere?

Reply:
we can make it a part of the API, like we did for iOS. So that if you want to use a custom font provider, it has to define what each of these (titleMedium, titleLarge, etc) is

Reply:
I was trying to avoid that since it means we need to override the theme for the paywall, instead of relying on the overall theme... but yeah, that might be the best way to make sure this can be customized. I will write up something so it can be part of the API.